src/styles/blocks/card.scss

```diff
@@ -0,0 +1,134 @@
+$button-color: #00acdc;
+$bg-color: white;
+$smalltext-color: #616070;
+$maintext-color: #060b35;
+
+* {
+  box-sizing: border-box;
+
+}
+
+h1,
+h2,
+p,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #d3d3d3;
+  background-color: $bg-color;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+
+  .card__img {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+    display: block;
+    background-image: url("../images/imac.jpeg");
+    background-size: cover;
+    margin-bottom: 0;
+    overflow: hidden;
+  }
+
+  .card__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $maintext-color;
+  }
+
+  .product-code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $smalltext-color;
+    width: 100%;
+  }
+
+  .reviews {
+    display: flex;
+    align-items: center;
+    margin-top: 16px;
+
+    .stars {
+      display: flex;
+    }
+
+    .stars__star {
+      background: url(../images/star.svg) no-repeat center;
+      width: 16px;
+      height: 16px;
+    }
+
+    @for $starCount from 1 through 4 {
+      .stars--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+
```

Comment:
use nesting, you have `stars` selector

Reply:
still not resolved

&__star etc..
![image](https://github.com/mate-academy/layout_catalog/assets/55200530/eb1e7df7-240f-474b-9448-221e0ff1cd91)